cypress-shared/integration/adyen/2.1-graphql_testcases.spec.js

```diff
@@ -0,0 +1,51 @@
+import {
+  graphql,
+  createAccountHolder,
+  validateCreateAccountHolderResponse,
+  refreshOnboarding,
+  validateRefreshOnboardingResponse,
+  closeAccountHolder,
+  validateCloseAccountHolderResponse,
+  sellers,
+  validateSellersResponse,
+  validateGetAdyenAccountResponse,
+  getAdyenAccount,
+  adyenAccountHolder,
+  validateAdyenAccountHolderResponse,
+} from '../../support/adyen/graphql_testcase'
+import { updateRetry, loginViaCookies } from '../../support/common/support'
+import { createAccount } from '../../support/adyen/outputvalidation'
+
+const {accountHolderCode,sellerId} = createAccount;
+const prefix = 'Graphql testcase'
+
+describe('Adyen GraphQL Validation', () => {
+  loginViaCookies()
+
+  it(`${prefix} - Create Account Holder`, updateRetry(2), () => {
+    graphql(
```

Comment:
@shashidhar85 
We have graphql testcases here
Along with post setup testcases we also need to verify UI here

Reply:
@saravananvenkatesan please check this